svc-aggregation/system/common.go

```diff
@@ -668,25 +759,28 @@ func createServerSearchIndex(computeSystem map[string]interface{}, oidKey, devic
 	}
 
 	// saving the firmware version
-	if firmwareVersion := getFirmwareVersiomn(oidKey); firmwareVersion != "" {
-		searchForm["FirmwareVersion"] = firmwareVersion
+	if !strings.Contains(oidKey, "/Storage") {
+		if firmwareVersion := getFirmwareVersiomn(oidKey); firmwareVersion != "" {
```

Comment:
change the function name `getFirmwareVersiomn` to `getFirmwareVersion`

Reply:
Corrected the func name